hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -558,6 +558,24 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_rv_dm_end_to_end_debug
+      desc: '''End to end debug test involving full GDB/OpenOCD/JTAG chain
+
+            - Load and begin executing some known binary
+            - Connect GDB via OpenOCD and JTAG to OpenTitan Earlgrey
+            - Trial the following activies within GDB, ensuring the correct
+              behaviour is seen:
+              - Interrupting and halting execution
+              - Setting and hitting breakpoints
```

Comment:
maybe specify the number of break points we support? it use to be only 1, but now its more I think?

Reply:
```suggestion
              - Setting, hitting, and deleting breakpoints using all available hardware breakpoints
```